hack/update-skaffold-deps.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
I think this script can be removed in favor of the Makefile target `update-skaffold-deps`.